src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1069,9 +1069,12 @@
     <Compile Include="$(CommonPath)Interop\Interop.ResultCode.cs">
       <Link>Common\Interop\Interop.ResultCode.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)Interop\Interop.TimeZoneInfo.cs">
+    <Compile Condition="'$(TargetsBrowser)' != 'true'" Include="$(CommonPath)Interop\Interop.TimeZoneInfo.cs">
```

Comment:
(nit) everywhere else in this file has the `Include` first, and the `Condition` second. I think we should match this convention, as it makes it easier to read when they all follow the same format.